openslides/core/static/js/core/core.js

```diff
@@ -605,6 +605,46 @@ angular.module('OpenSlidesApp.core.site', [
             }
         );
     };
+
+    // *** countdown ***
+    $scope.countdowns = [];
+    $scope.messages = [];
+    _.forEach(Projector.get(1).elements, function(element) {
+        if (element.name == 'core/countdown') {
+            $scope.countdowns.push(element);
+        }
```

Comment:
What about not using an array but an object. In the forEach loop you can also get access to the dict key and so use it.